seven_days_average/seven_days_average.py

```diff
@@ -0,0 +1,83 @@
+import csv
+import requests
+
+NUMBER_OF_TRACKED_DAYS = 14 
+def main():
+
+    # Read NYTimes Covid Database
+    request = requests.get(
+        "https://raw.githubusercontent.com/nytimes/covid-19-data/master/us-states.csv"
+    )
+    decoded_content = request.content.decode("utf-8")
+    file = decoded_content.splitlines()
+    reader = csv.DictReader(file)
+
+    # Construct 14 day lists of new cases for each states
+    new_cases = calculate(reader, NUMBER_OF_TRACKED_DAYS)
+
+    print("Choose one or more states to view average COVID cases.")
+    print("Press enter when done.\n")
+
+    state = input("State: ")
+    print(f"\nSeven-Day Averages")
+
+    print_result(state, new_cases, NUMBER_OF_TRACKED_DAYS)
+
+
+def calculate(reader, NUMBER_OF_TRACKED_DAYS):
```

Comment:
in this function number of tracked days should be lowercased, here it's a function argument, not the constant value